test/retry.test.ts

```diff
@@ -262,4 +266,87 @@ describe("Automatic Retries", function () {
 
     expect(caught).toEqual(testStatuses.length);
   });
+
+  it("Should use default retry options if no options are passed to the plugin", async function () {
```

Comment:
One caveat about this test case is that I'm not sure how to shorten the exponential back retry delay, since I can't pass a small `retryAfterBaseValue` to `retry` otherwise it won't be testing the default parameter functionality that I'm trying to cover with this test case.

I set up four requests so that the test asserts that the default number of retries happens (3) before failing, but that means it takes about 14 seconds for the test case to run with the default exponential backoff.

A slightly more unit test-y approach I can take is to call `retry` with an octokit instance and no options and assert that no TypeErrors are thrown due to trying to access properties on an `undefined` options argument inside the `retry` function.

Reply:
I'd rather have slow tests than incomplete tests :) Thank you so much for taking that on! I'd say we worry about the coverage first, and then look into making the tests run faster. But maybe it's not a problem, it's a very small library that does one thing, I think the impact of slow tests is neglect-able and well contained within this plugin